www/assets/css/about.scss

```diff
@@ -220,12 +220,11 @@
     }
 
     li:before {
-      content: "remove";
```

Comment:
@pdpinch Default html bullet was replaced by `remove material icon` here. Maybe due to `content: "remove"` "remove" is announced, but I'm not very sure since I can not reproduce the issue. 
I'm using this [screen reader](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) and it does not announce "remove". Maybe some other screen readers announce the CSS content.

Anyways, I have done this workaround. Can you please get this thing tested and let me know if this or any similar issue still occurs?

Reply:
Ok. Let's merge this and we'll see how it works.

Reply:
Sure

Reply:
@pdpinch I tried some other screen readers too but they don't seem to announce the CSS content but if "remove" was being announced previously, then I have a feeling that "-" (dash) might be announced this time.
Let me know once our developer tests it